packages/generator/src/sdk-metadata/links.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { executeHttpRequest } from '@sap-cloud-sdk/core';
+import { ErrorWithCause } from '@sap-cloud-sdk/util';
+import { getLinks } from './links';
+
+describe('sdk metadata links',()=>{
```

Comment:
This is an e2e test, 10 sec is still fine. If this becomes longer or unstable, we might want to move to nightly job.

Reply:
I had the same thought - I add an [E2E] to all tests making calls to the internet.